Symfony/src/AppBundle/Resources/views/Tier/index.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
+
+{% block stylesheets %}
+    <!-- Custom CSS -->
+    {% include 'AppBundle:Tier:css.html.twig' %}
+{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-10 col-xs-offset-1">
+                {% for tier in tiers %}
+                    <div class="col-sm-4">
+                        <div class="panel price
+                            {% if tier.id == 1 %}
+                                panel-grey
+                            {% elseif tier.id == 2 %}
+                                panel-blue
+                            {% elseif tier.id == 3 %}
+                                panel-red
+                            {% endif %}
+                            ">
+                            <div class="panel-heading  text-center">
+                                <h3>{{ tier.tierName|upper }} PLAN</h3>
+                            </div>
+                            <div class="panel-body text-center">
+                                <p class="lead" style="font-size:40px"><strong>${{ tier.price }} / month</strong></p>
```

Comment:
This hardcoded dollar sign before the price will be difficult to change in the future if l20n is needed. Instead we could have used a [twig currency filter](http://twig.sensiolabs.org/doc/extensions/intl.html#localizedcurrency) that will make it a breeze to update the code in the future.